modules/manta/germline/main.nf

```diff
@@ -26,13 +26,14 @@ process MANTA_GERMLINE {
     script:
     def args = task.ext.args ?: ''
     def prefix = task.ext.prefix ?: "${meta.id}"
-    def options_manta = target_bed ? "--exome --callRegions $target_bed" : ""
+    def options_manta = target_bed ? "--callRegions $target_bed" : ""
```

Comment:
Why removing the --exome?
Or this is now something that should be done with the `ext.args`?

Reply:
This allows us to use callRegions but for example using interval files for WGS. --exome then via ext.args if the data is actual some type of panel